src/indexer/services/coinmarketcap/coinmarketcap.service.ts

```diff
@@ -35,12 +35,12 @@ class CoinMarketCapService {
   }
 
   private async getValueConvertion(amount: string, tokenSymbol: string): Promise<CoinMaketCapResponse["quote"]["USD"]["price"]> {
-    const tokenValue: string | undefined = await this.memoryCache.get(tokenSymbol)
+    const tokenValue: number | undefined = await this.memoryCache.get(tokenSymbol)
     if (tokenValue) {
-      return BigNumber(amount).times(BigNumber(tokenValue))
+      return Number(amount) * tokenValue
```

Comment:
Why did we change this from BigNumber

Reply:
I will revert the change. The coinmarketcap returns the value of type number, that's why I changed everything into number. But then again, the amount can be BigNumber...